server/app.ts

```diff
@@ -62,6 +63,7 @@ export default function createApp(services: Services): express.Application {
   app.use('/product-dependencies', productDependencyRoutes(services))
   app.use('/missing-from-catalogue', missingFromCatalogueRoutes(services))
   app.use('/namespaces', namespacesRoutes(services))
+  app.use('/githubreporequestform', formsRoutes(services))
```

Comment:
**suggestion:** can we call this `/new-github-repo-request-form`?

Reply:
Updated   app.use('/forms', formsRoutes(services)) similar to reports